src/lib/requests.ts

```diff
@@ -45,13 +45,15 @@ import {
 const configRequest: BaseRequestConfigType = {
   dfspId: config.get('PARTICIPANT_ID') as string,
   logger: Logger,
-  // TODO: Decide on below later - Handled in future ticket #361
-  // Also decide on need for jwsSigningKey
-  jwsSign: false,
+  jwsSign: config.get('JWS_SIGN') as boolean,
+  // TODO: Confirm if this is the correct way to invoke JWS SIGNING
+  //        Is type provided Buffer?
+  jwsSigningKey: config.get('JWS_SIGNING_KEY') as unknown as Buffer,
```

Comment:
type casting isn't invoking the constructor 
try: https://nodejs.org/api/buffer.html#buffer_static_method_buffer_from_string_encoding